test/matrixFunctionality.cpp

```diff
@@ -233,49 +216,23 @@ TEST_P(MultipleOfTwoMatrix, TheMultipleEqualOperatorShouldSetFirstMatrixEqualToT
 	EXPECT_EQ(FIRST_MATRIX, THIRD_MATRIX);
 }
 
-INSTANTIATE_TEST_SUITE_P(
-		MultipleData,
-		MultipleOfTwoMatrix,
-		Values(
-				std::make_tuple(
-						Matrix<int>({{1, 2}, {1, 2}}),
-						Matrix<int>({{1, 2}, {1, 2}}),
-						Matrix<int>({{3, 6}, {3, 6}})
-				),
-				std::make_tuple(
-						Matrix<int>({{1, 2, 3, 4, 5, 6, 7}}),
-						Matrix<int>({{7}, {6}, {5}, {4}, {3}, {2}, {1}}),
-						Matrix<int>({{84, 0, 0, 0, 0, 0, 0}})
-				),
-				std::make_tuple(
-						Matrix<int>({
-								{1, 2, 3, 4, 5, 6, 7},
-								{1, 2, 3, 4, 5, 6, 7},
-								{1, 2, 3, 4, 5, 6, 7},
-								{1, 2, 3, 4, 5, 6, 7},
-								{1, 2, 3, 4, 5, 6, 7}
-						}),
-						Matrix<int>({
-								{7, 2, 3, 4, 5, 6, 9, 7},
-								{6, 2, 3, 4, 5, 6, 9, 7},
-								{5, 2, 3, 4, 5, 6, 9, 7},
-								{4, 2, 3, 4, 5, 6, 9, 7},
-								{3, 2, 3, 4, 5, 6, 9, 7},
-								{2, 2, 3, 4, 5, 6, 9, 7},
-								{1, 2, 3, 4, 5, 6, 9, 7}
-						}),
-						Matrix<int>({
-								{84, 56, 84, 112, 140, 168, 252},
-								{84, 56, 84, 112, 140, 168, 252},
-								{84, 56, 84, 112, 140, 168, 252},
-								{84, 56, 84, 112, 140, 168, 252},
-								{84, 56, 84, 112, 140, 168, 252}
-						})
-				)
-		)
-);
-
-class OppositeOfMatrix: public ::testing::TestWithParam<std::tuple<Matrix<int>, Matrix<int>>> {};
+INSTANTIATE_TEST_SUITE_P(MultipleData, MultipleOfTwoMatrix,
+		Values(std::make_tuple(Matrix<int>({{1, 2}, {1, 2}}), Matrix<int>({{1, 2}, {1, 2}}),
+					   Matrix<int>({{3, 6}, {3, 6}})),
+				std::make_tuple(Matrix<int>({{1, 2, 3, 4, 5, 6, 7}}), Matrix<int>({{7}, {6}, {5}, {4}, {3}, {2}, {1}}),
+						Matrix<int>({{84, 0, 0, 0, 0, 0, 0}})),
+				std::make_tuple(Matrix<int>({{1, 2, 3, 4, 5, 6, 7}, {1, 2, 3, 4, 5, 6, 7}, {1, 2, 3, 4, 5, 6, 7},
```

Comment:
Can we say to clang-format each nested array in a new line if need break?

Reply:
i do not know